1080i/SkinSettings.xml

```diff
@@ -798,12 +806,12 @@
                         <onclick>Skin.ToggleSetting(DisableClock)</onclick>
                     </control>
 
-                    <control type="radiobutton" id="19709" description="Clearlogo">
+                    <control type="radiobutton" id="19710" description="Show clearart">
                         <visible>Integer.IsEqual(Window.Property(CurrentFocus),9007)</visible>
                         <include>Defs_Settings_Button</include>
```

Comment:
Move this setting out of the "Furniture" section and into the "Video / Music" section where the rest of the OSD settings are. The visibility condition will need to change from 9007 to 9005 and the id will need to be changed to one starting with id="195XX" (instead of 197XX).

Reply:
Ok, thanks. You're right, I didn't think about about the overlap when info is on, I always have it off.
Thanks for your patience.